### DISCUSSION TITLE
Release new version with strict server generation

### DISCUSSION THREAD

**Discussion by namp10010** [oapi-codegen/oapi-codegen#677] [General]
Hi,

Thanks for creating this awesome tool.

I am really keen to use the strict server generation feature which has been added after the latest release. Do you have plan for a new release will come to live with this feature 🙏 ?

**Comment by jamietanna** [comment]
Hey, you should be able to pin to the source version of the library i.e.

```sh
go get github.com/deepmap/oapi-codegen@HEAD
```

In the meantime - I think there are still a couple of things we'll want to finalise before this next release

**Reply by namp10010** [reply]
thanks, will pin the version for now.

**Reply by yk-eukarya** [reply]
@jamietanna 
Is there any schedule for the next release?

**Reply by jamietanna** [reply]
https://github.com/deepmap/oapi-codegen/discussions/784 :)
